docs/reference/search/search-your-data/retrieve-selected-fields.asciidoc

```diff
@@ -6,72 +6,46 @@
-
-A document's `_source` is stored as a single field in Lucene. So the whole
-`_source` object must be loaded and parsed even if only a small number of
-fields are requested. To avoid this limitation, you can try another option for
-loading fields:
-
-* Use the <<docvalue-fields, `docvalue_fields`>>
-parameter to get values for selected fields. This can be a good
-choice when returning a fairly small number of fields that support doc values,
-such as keywords and dates.
-* Use the <<request-body-search-stored-fields, `stored_fields`>> parameter to
-get the values for specific stored fields (fields that use the
-<<mapping-store,`store`>> mapping option).
-
-You can also use the <<script-fields,`script_field`>> parameter to transform
-field values in the response using a script.
-
-You can find more detailed information on each of these methods in the
-following sections:
+* Use the <<search-fields-param,`fields` option>> to extract the values of
+fields present in the index mapping
+* Use the <<source-filtering,`_source` option>> if you need to access the original data that was passed at index time
 
-* <<search-fields-param>>
-* <<docvalue-fields>>
-* <<stored-fields>>
-* <<source-filtering>>
-* <<script-fields>>
+You can use both of these methods, though the `fields` option is preferred
+because it consults both the document data and index mappings. In some
+instances, you might want to use <<field-retrieval-methods,other methods>> of
+retrieving data.
 
 [discrete]
 [[search-fields-param]]
-=== Fields
-
+=== The `fields` option
 // tag::fields-param-desc[]
-The `fields` parameter allows for retrieving a list of document fields in
-the search response. It consults both the document `_source` and the index
-mappings to return each value in a standardized way that matches its mapping
-type. By default, date fields are formatted according to the
-<<mapping-date-format,date format>> parameter in their mappings. You can also
-use the `fields` parameter to retrieve <<runtime-retrieving-fields,runtime field
-values>>.
+To retrieve specific fields in the search response, use the `fields` parameter.
+Because it accesses the index mappings, the `fields` parameter provides several
+advantages over referencing the `_source` directly. Specifically, the `fields`
+parameter:
+
+* Returns each value in a standardized way that matches its mapping type
+* Accepts <<multi-fields,multi-fields>> and <<alias,field aliases>>
+* Formats date fields by default according to the
+<<mapping-date-format,date format>> parameter in their mappings
+* Retrieves <<runtime-retrieving-fields,runtime field values>>
+* Returns fields calculated by a script at index time
 // end::fields-param-desc[]
 
+Other mapping options are also respected, including
+<<ignore-above,`ignore_above`>>, <<ignore-malformed,`ignore_malformed`>> and
+<<null-value,`null_value`>>.
+
+When retrieving standard fields, the `fields` option returns values in the way
```

Comment:
It feels a little confusing how the first and second sentences relate to each other, and also what "returns values from `_source` exactly how they are indexed" means. Suggested reworking: "The `fields` option returns values in the way that matches how Elasticsearch indexes them. For standard fields, the `fields` option accomplishes this by looking in `_source` to find the values, then parsing and formatting them using the mappings."